test/integration/auth.spec.ts

```diff
@@ -520,8 +520,12 @@ describe('admin.auth', () => {
       const rawPassword = 'password';
       const rawSalt = 'NaCl';
 
-      before(() => {
-        tenantAwareAuth = admin.auth().tenantManager().authForTenant(createdTenantId);
+      before(function() {
+        if (!createdTenantId) {
+          this.skip();
```

Comment:
When does this happen? if the tenant creation fails in an earlier test?

Reply:
It seems the skip test is not getting applied for nested tests. So i added a check when no tenant is created (typically when the test is skipped or if tenant creation fails) to skip the nested tests.